resources/dicts/thoughts/alive/general.json

```
@@ -51,87 +49,1497 @@
-            "Feeling confined and cooped up",
-            "Just wants to sleep",
-            "Wonders why recovering is so exhausting",
-            "Basking in the sun at the medicine den entrance",
-            "Feeling frustrated at {PRONOUN/m_c/poss} own exhaustion",
-            "Doesn't think {PRONOUN/m_c/subject} should have to sleep in the medicine den",
-            "Eats some prey because {PRONOUN/m_c/subject} know {PRONOUN/m_c/subject} should, not because of hunger",
-            "Stands up for a second and feels lightheaded",
-            "Isn't having a fun time"
-        ]
+            "Is urging r_c to rest",
+            "Brought r_c some prey",
+            "Wonders if {PRONOUN/m_c/subject}'ll still be accepted into StarClan after taking r_c as a mate",
+            "Hopes r_c will still be accepted into StarClan",
+            "Hopes StarClan will forgive {PRONOUN/m_c/object} and r_c for becoming mates",
+            "Offers to help r_c with {PRONOUN/r_c/poss} patients"
+        ],
+        "main_status_constraint": [
+            "warrior",
+            "medicine cat",
+            "mediator",
+            "deputy",
+            "leader",
+            "elder"
+        ],
+        "random_status_constraint": [
+            "medicine cat"
+        ],
+        "random_living_status": [
+            "alive"
+        ],
+        "relationship_constraint": [
+			"mates"
+		]
     },
     {
-        "id": "gen_alive_not_working_insteadinthemedden_withmedcat",
-        "not_working": true,
+        "id": "gen_sibling",
         "thoughts": [
-            "Watches r_c fuss around the herb stores, half-asleep",
-            "Isn't going to argue with r_c about {PRONOUN/m_c/poss} medical care",
-            "Argues with r_c about {PRONOUN/m_c/poss} medical care",
-            "Dutifully stretching out {PRONOUN/m_c/poss} limbs like r_c told {PRONOUN/m_c/object} to",
-            "Asks r_c for pain relief",
-            "Wishes {PRONOUN/m_c/subject} understood what r_c is doing"
+            "Is sharing prey with r_c",
+            "Took a nap with r_c",
+            "Took a nap in r_c's nest",
+            "Wants to work hard to impress r_c",
+            "Had a pointless, petty argument with r_c",
+            "Startled r_c to get back at {PRONOUN/r_c/object} for something or other",
+            "Casually insulted r_c in passing"
         ],
-        "random_status_constraint": [
-            "medicine cat"
+        "main_status_constraint": [
+            "any"
         ],
         "random_living_status": [
-            "living"
+            "alive"
+        ],
+        "relationship_constraint": [
+			"siblings"
+		]
+    },
+    {
+        "id": "gen_sibling_adult",
+        "thoughts": [
+            "Play fought with r_c just like old times",
+            "Reminisced with r_c on when they were kits",
+            "Took r_c's fresh_kill {PRONOUN/r_c/subject} were saving because {PRONOUN/r_c/subject} were saving it", 
```

```suggestion
            "Took r_c's fresh_kill that {PRONOUN/r_c/subject} were saving", 
```
Gets a little confusing, I think it's clearer without the last bit

---

I actually worded it that way because its meant to be them intentinally taking the prey specifically because they knew their sibling was saving it. Will reword it to be more clearer

---

Wording hopefully better now